sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar.projectVersion=1.0.0-RC8
```

Comment:
should be replaced by the next version ?
Or can it be dynamically generated from the package.json's `version` attribute ?